demo/index.html

```diff
@@ -16,10 +16,10 @@
 
   <title>quick-element</title>
 
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
We deliberately avoid referencing dependencies this way. We always assume that dependencies are installed the way they would be installed when `bower` installs them. In other words, each dependency is assumed to be in a sibling folder of the current project's root folder.